doc/quick-start.md

```diff
@@ -41,53 +43,59 @@ Load the extension:
 %load_ext sql
 ```
 
-Let's see an example using a SQLite database. Let's insert some sample data from the TIOBE index:
+Let's download some sample `.csv` data:
 
 ```{code-cell} ipython3
-%%sql sqlite://
-CREATE TABLE languages (name, rating, change);
-INSERT INTO languages VALUES ('Python', 14.44, 2.48);
-INSERT INTO languages VALUES ('C', 13.13, 1.50);
-INSERT INTO languages VALUES ('Java', 11.59, 0.40);
-INSERT INTO languages VALUES ('C++', 10.00, 1.98);
+from pathlib import Path
+from urllib.request import urlretrieve
+
+if not Path("penguins.csv").is_file():
+    urlretrieve("https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv",
+                "penguins.csv")
 ```
 
-## Querying
+Start a DuckDB in-memory database:
 
 ```{code-cell} ipython3
-%sql SELECT * FROM languages
+%sql duckdb://
 ```
 
-## `pandas` integration
+## Querying
 
 ```{code-cell} ipython3
-result = %sql SELECT * FROM languages
+%sql SELECT * FROM penguins.csv LIMIT 3
```

Comment:
I'd also add a sample here of the magics cell `%%sql`

Reply:
fixed it